src/cr/cube/dimension.py

```diff
@@ -954,6 +959,41 @@ def _order_dict(self):
         return self._dimension_transforms_dict.get("order") or {}
 
 
+class _PairwiseSignificanceSpec(object):
+    """Value object providing access to details of pairwise significance information."""
+
+    def __init__(self, dimension, dimension_transforms_dict):
+        self._dimension = dimension
+        self._dimension_transforms_dict = dimension_transforms_dict
+
+    @lazyproperty
+    def column_idx(self):
+        """int index of base column as reference for the pairwise sig tests."""
+        return self._pairwise_sig_dict.get("column_significance", {}).get("column_idx")
+
+    @lazyproperty
+    def insertion_idx(self):
+        """int index of insertion column as reference for the pairwise sig tests."""
+        if self._insertion_id:
+            return self._dimension.insertion_ids.index(self._insertion_id)
+        return None
+
+    @lazyproperty
+    def _insertion_id(self):
+        """int insertion-id in the "insertion_id" field of the transform dict."""
```

Comment:
Clarify this docstring. It's also a restatement of where to get it. Make it say what it _means_ instead. I think it identifies the reference column when the reference column is an insertion rather than a "base" column.